RetroRabbit/RGO-Client
# login page styled with material dashboard

- Added Angular Project
- login implementation
- Added routing from sign in to home page
- login testing
- testing
- login with google works. no state management
- Apply suggestions from code review
- Merged PR 4809: angular login with google
- called api to check user exists
- Merge branch 'Frontend-DEV' into feature/connect-to-api
- added response type and moved the piping to the service
- state management initial setup
- testing state
- Added state management
- Merge branch 'Frontend-DEV' into feature/stateManagement
- Merged PR 4813: Login
- Merge branch 'Frontend-DEV' into feature/stateManagement
- Added material dashboard
- Merged PR 4833: Implementing state manangement
- Merged PR 4835: Added material dashboard
- login page styled with material dashboard
